### PR TITLE
Fixed issue 258

### PR DESCRIPTION
**Pull request by FelixWeik**
Data frame asks for directory paths now instead of filenames

### CODE REVIEW

File: src/osm_configurator/view/toplevelframes/data_frame.py [hunk 1]
```diff
@@ -294,7 +295,7 @@ def __select_cut_out(self):
         """
         Opens the explorer letting the user choose a file selecting the cut-out
         """
-        chosen_path: Path = self.__open_explorer(list["png"])  # TODO: insert accepted filetypes
+        chosen_path: Path = self.__get_directory_path()
```
Comment (GloeStoe) [inline src/osm_configurator/view/toplevelframes/data_frame.py:298]: Why do you ask the user for a directory when selecting a cut out file? It was correct, the way it was before

File: src/osm_configurator/view/toplevelframes/data_frame.py [hunk 2]
```diff
@@ -312,11 +313,11 @@ def __select_osm_data(self):
         """
         Opens the explorer letting the user choose a file selecting the osm-data
         """
-        chosen_path: Path = self.__open_explorer(None)  # TODO: insert accepted filetypes
+        chosen_path: Path = self.__get_directory_path()
```
Comment (GloeStoe) [inline src/osm_configurator/view/toplevelframes/data_frame.py:316]: Why do you ask the user for a directory when selecting a osm file?

**Review by GloeStoe — CHANGES_REQUESTED** [review]
The user should enter a directory when selecting a category import, if he selects OSM and geojson data he must enter osm and geojson files respectively

**Review by GloeStoe — APPROVED** [review]

**Review by ujrlh — APPROVED** [review]
looks good and works

File: src/osm_configurator/view/toplevelframes/data_frame.py [hunk 3]
```diff
@@ -296,8 +296,8 @@ def __select_cut_out(self):
         Opens the explorer letting the user choose a file selecting the cut-out
         """
         chosen_path: Path = self.__get_file_path()
-
-        if not chosen_path.exists():
+        print(chosen_path)
```
Comment (ujrlh) [inline src/osm_configurator/view/toplevelframes/data_frame.py:301]: remove print statement

**Pull request merged**